demisto_sdk/commands/common/constants.py

```diff
@@ -1151,7 +1151,7 @@ class PB_Status:
 
 ENDPOINT_FLEXIBLE_REQUIRED_ARGS = ["ip", "id", "hostname"]
 
-GENERIC_COMMANDS_NAMES = BANG_COMMAND_NAMES.union({'send-mail', 'send-notification', 'cve-latest', 'cve-search'})
+GENERIC_COMMANDS_NAMES = BANG_COMMAND_NAMES.union({'send-mail', 'send-notification', 'cve-latest', 'cve-search', 'search'})
```

Comment:
it will affect id-set existing flow, by design?

Reply:
It seems like the `search` commands are not being collected in the dependencies, but I don't find where it's stated.
I think it's safe to add it to GENERIC_COMMAND_NAMES beacuse it's also a generic command and should be ignored.